servicex/config_default.yaml

```diff
@@ -5,8 +5,9 @@ api_endpoints:
   - endpoint: http://localhost:5000
     # token: xxx
 
-# This is the path of the cache. The "/tmp" will be translated, platform appropriate.
-cache_path: /tmp/servicex
+# This is the path of the cache. The "/tmp" will be translated, platform appropriate, and
+# the env variable USER will be replaced.
+cache_path: /tmp/servicex_${USER}
```

Comment:
Another option is `.../servicex/${USER}`, to avoid cluttering the parent directory if ServiceX is being used by a large number of users on the same machine.

Reply:
I thought of that, but that means whoever creates the `/tmp/servicex` directory has to get the permissions right - so we are back with the same issue that started this.